src/ctia/stores/es/verdict.clj

```diff
@@ -0,0 +1,8 @@
+(ns ctia.stores.es.verdict
+  (:require [ctia.stores.es.crud :as crud]
+            [ctim.schemas.verdict :refer [StoredVerdict]]))
+
+(def handle-create-verdict (crud/handle-create :verdict StoredVerdict))
+(def handle-read-verdict (crud/handle-read :verdict StoredVerdict))
+(def handle-delete-verdict (crud/handle-delete :verdict StoredVerdict))
+(def handle-list-verdicts (crud/handle-find :verdict StoredVerdict))
```

Comment:
I am really impressed at how this, the IVerdictStore protocol extension in stores.es.store, and the mapping in stores.es.mapping are enough to implement Verdicts in Elastic Search. Other than saving a lot of work, it's a great way to know that a new data entity will work correctly.